eng/common/pipelines/templates/steps/update-docsms-metadata.yml

```diff
@@ -26,6 +26,9 @@ parameters:
     type: object
     default:
       - '**'
+  - name: PipelineRequestBy
+    type: string
+    default:  $(Build.RequestedFor) 
```

Comment:
Have we tested to see what this value ends up looking like? For example is it DevOps or Github identity?